.github/workflows/kani.yml

```diff
@@ -156,6 +156,7 @@ jobs:
             docker run -w /tmp/kani/tests/cargo-kani/simple-lib  $tag  cargo kani
```

Comment:
These should also be a separate script.

Reply:
I'll do this one as well in a follow-up PR.